### PR TITLE
[zigbee2mqtt] update to use common library

### PR DESCRIPTION
**Pull request by onedr0p**
#### Special notes for your reviewer:

Not sure on a upgrade path @bjw-s since you are using this now you can give you input if needed.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

**Comment by onedr0p** [comment]
Chart is failing because it cannot find the zigbee device when the container starts. I think we might have to merge this as is :/

File: charts/zigbee2mqtt/Chart.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-description: Bridges events and allows you to control your Zigbee devices via MQTT
 appVersion: 1.15.0
-version: 2.1.0
+description: Bridges events and allows you to control your Zigbee devices via MQTT
+name: zigbee2mqtt
+version: 3.0.0
 keywords:
   - zigbee
   - mqtt
+  - home-assistant
```
Comment (bjw-s) [inline charts/zigbee2mqtt/Chart.yaml:9]: Maybe replace home-assistant with home-automation? It's not really that tightly coupled with HA.

File: charts/zigbee2mqtt/OWNERS [hunk 2]
```diff
@@ -0,0 +1,4 @@
+approvers:
+- billimek
+reviewers:
+- billimek
```
Comment (bjw-s) [inline charts/zigbee2mqtt/OWNERS:4]: Missing newline

File: charts/zigbee2mqtt/templates/common.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+subPath: configuration.yaml
+{{- end -}}
+
+{{- $volumeMount := include "zigbee2mqtt.configmap.volumeMount" . | fromYaml -}}
+{{- if $volumeMount -}}
+    {{- $additionalVolumeMounts := append .Values.additionalVolumeMounts $volumeMount }}
+    {{- $_ := set .Values "additionalVolumeMounts" (deepCopy $additionalVolumeMounts) -}}
+{{- end -}}
+
+{{/* Render the templates */}}
```
Comment (bjw-s) [inline charts/zigbee2mqtt/templates/common.yaml:30]: Couldn't have done it better myself 😜

File: charts/zigbee2mqtt/values.yaml [hunk 4]
```diff
@@ -1,16 +1,64 @@
 # Default values for zigbee2mqtt.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
 
 image:
   repository: koenkk/zigbee2mqtt
   pullPolicy: IfNotPresent
-  # Overrides the image tag whose default is the chart appVersion.
-  tag: ""
+  tag: 1.14.4
```
Comment (bjw-s) [inline charts/zigbee2mqtt/values.yaml:6]:
```suggestion
  tag: 1.16.1
```

**Review by bjw-s — CHANGES_REQUESTED** [review]
Some small things... will take a look to see if I can make it start up without a device being present

File: charts/zigbee2mqtt/Chart.yaml [hunk 5]
```diff
@@ -1,12 +1,12 @@
 apiVersion: v2
-name: zigbee2mqtt
-type: application
-description: Bridges events and allows you to control your Zigbee devices via MQTT
 appVersion: 1.15.0
```
Comment (bjw-s) [inline charts/zigbee2mqtt/Chart.yaml:2]:
```suggestion
appVersion: 1.16.1
```

**Comment by bjw-s** [comment]
Previous version of the chart was failing with the same error it seems: https://github.com/k8s-at-home/charts/pull/99/checks

File: charts/zigbee2mqtt/Chart.yaml (hunk 1 above)
Comment (onedr0p) [reply charts/zigbee2mqtt/Chart.yaml:9]: i think home-assistant is fine, it's related and in the zwave chart too :)

**Review by bjw-s — APPROVED** [review]
Let's merge this even though the app code is still failing, it seems impossible to bypass the check for a serial device

**Comment by onedr0p** [comment]
Yep, gonna go ahead and merge this.

**Pull request merged**